web-client/integration-tests/docketClerkAddsAutomaticBlockCaseToTrial.test.js

```diff
@@ -34,7 +34,7 @@ describe('Adds automatic block case to trial', () => {
   };
 
   loginAs(cerebralTest, 'petitionsclerk@example.com');
-  petitionsClerkCreatesNewCase(cerebralTest, fakeFile, trialLocation);
+  petitionsClerkCreatesNewCase(cerebralTest, true, { trialLocation });
```

Comment:
Nit: Could the second argument be a named one? Hard to tell what `true` is being passed in for and I almost thought it was a typo when removing `fakeFIle` :D

Reply:
Yes good feedback. I'll do that!